src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2611,6 +2611,72 @@ Task Synchronization API can be used for multiple dynamic synchronization patter
 
 A complete example of Producer / Consumer, _ServiceDeployerConsumer_ can be downloaded link:examples/ServiceDeployerConsumer.xml[here].
 
+[[_task_signal_api]]
+=== Signal API
+
+The *Signal API* is a high-level API that enables users to interact with running jobs via signals (we also call them actions).
+That is, it allows a job to expose specific actions (e.g., soft stop, scale up, scale down) that can triggered by users from the *Workflow Execution portal*.
+
+More specifically, the Signal API allows a workflow to:
+
+  . expose signals, i.e., display them in the Workflow Execution portal (e.g., display the `soft_stop` signal)
+
+  . wait to receive these signals, or check for their reception regularly, and
+
+  . execute the corresponding behavior (defined in the workflow) at the signal reception.
+
+The Signal API uses the synchronization API to provide a *persistent key-value channel* to store job signals.
```

Comment:
The synchronization API has to be referenced

Reply:
Done